### PR TITLE
bump to kube v0.1.0

### PR DESCRIPTION
**Pull request by bacongobbler**
closes https://github.com/spinkube/spin-plugin-kube/issues/43

### CODE REVIEW

File: manifests/kube/kube@0.0.1.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "kube",
+  "description": "A plugin to manage spin apps",
```
Comment (karthik2804) [inline manifests/kube/kube@0.0.1.json:3]: Do we want the  descriptions to match?

Comment (bacongobbler) [reply manifests/kube/kube@0.0.1.json:3]: I’m okay with having the descriptions to match. However I’m also happy to leave it as-is for historical purposes.

File: manifests/kube/kube.json [hunk 2]
```diff
@@ -1,40 +1,46 @@
 {
   "name": "kube",
-  "description": "A plugin to manage spin apps",
+  "description": "Commands for publishing applications to Kubernetes",
```
Comment (karthik2804) [inline manifests/kube/kube.json:3]: Also would something along the lines of "A plugin to manage Spin apps in Kubernetes" be better?

File: manifests/kube/kube@0.0.1.json (hunk 1 above)
Comment (karthik2804) [reply manifests/kube/kube@0.0.1.json:3]: I am good with either as well.

File: manifests/kube/kube.json [hunk 3]
```diff
@@ -1,40 +1,46 @@
...
   "homepage": "https://github.com/spinkube/spin-plugin-kube",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "spinCompatibility": ">=1.5",
```
Comment (bacongobbler) [inline manifests/kube/kube.json:6]: @karthik2804 wanted to know if we’ve actually tested this plugin against Spin 1.5.

@rajatjindal have you tested this plugin against spin 1.5? If not we should update this to a more recent version.

Comment (rajatjindal) [reply manifests/kube/kube.json:6]: I've not tested this with spin 1.5. we should change it to 2.3.1 as we have a OCI push fix in that version?

Comment (rajatjindal) [reply manifests/kube/kube.json]: I've pushed a commit to have min spin version 2.3.1

File: manifests/kube/kube.json (hunk 2 above)
Comment (macolso) [reply manifests/kube/kube.json:3]: Or even a bit more descriptive, "A plugin to manage Spin apps as custom resources in Kubernetes"?

Comment (bacongobbler) [reply manifests/kube/kube.json:3]: I'd prefer to leave out the "custom resources" part in the description. The intent of the plugin is to manage spin apps in Kubernetes. Custom resources is how we do that today, but it doesn't always have to be that way.

File: manifests/kube/kube@0.0.1.json (hunk 1 above)
Comment (bacongobbler) [reply manifests/kube/kube@0.0.1.json:3]: I'll leave it as-is for historical purposes

File: manifests/kube/kube.json (hunk 2 above)
Comment (bacongobbler) [reply manifests/kube/kube.json]: updated.

**Review by karthik2804 — APPROVED** [review]

**Pull request merged**